torchvision/models/quantization/mobilenetv3.py

```diff
@@ -194,18 +194,33 @@ def mobilenet_v3_large(
     **kwargs: Any,
 ) -> QuantizableMobileNetV3:
     """
-    Constructs a MobileNetV3 Large architecture from
-    `"Searching for MobileNetV3" <https://arxiv.org/abs/1905.02244>`_.
+    MobileNetV3 model from
```

Comment:
Why did you make this change? Clarifying that this is the variant ("Large" in this case) is something we do in all similar docs of models.